dotCMS/src/main/java/com/dotcms/variant/business/web/CurrentVariantSessionItem.java

```diff
@@ -0,0 +1,55 @@
+
+package com.dotcms.variant.business.web;
+
+import java.time.Instant;
+import java.util.Objects;
+
+/**
+ * Represents a session item for the current variant.
+ * This is a Flash Session Item that will be available for 10 seconds.
+ */
+public class CurrentVariantSessionItem {
+
+    private final int SECONDS_EXPIRE_TIME  = 10;
+
+    private Instant addedDate;
+    private String variantName;
```

Comment:
those should be final

Reply:
done https://github.com/dotCMS/core/pull/25387/commits/d20b65b5f6684525115757a4a73bf99fa6e59a9e#diff-93e02c67ffc61213dd2dab902128bbfce57006d11ccbb366f8e1452b366461d0R16-R17